settings/Localization.setting.php

```diff
@@ -218,11 +218,11 @@
     'quick_form_type' => 'YesNo',
     'default' => '0',
     'add' => '4.3',
-    'title' => ts('Inherit CMS Language'),
+    'title' => ts('Inherit User Language'),
```

Comment:
@ufundo I don't like changing this because for WP/Joomla/BackDrop/Drupal8 we are inheriting from the CMS not the user setting (User setting may affect what the CMS sets)

Reply:
@mlutfy what do you think about the language changes here?

Reply:
Ah. I was thinking it was the User either way (Standalone user or CMS user)... hadn't thought the CMS could decide on language another way. 

"Inherit CMS/User Language"?

I think there's a few places like this where the ideal would be to override the wording on Standalone - I think that *should* be possible using a hook on settings meta, but I haven't quite pinned it down.

Reply:
Have updated to use `alterSettingsMetaData` hook in standaloneusers, so the wording is only changed on Standalone :+1: